filter-readme-embedded-graphics.sed

```diff
@@ -4,3 +4,8 @@
 s#\[(.+)\]\(\.\/(.*)\)#[\1](https://docs.rs/embedded-graphics/latest/embedded_graphics/\2)#g
 s#\[(.+)\]\: +\.\/(.*)#[\1]: https://docs.rs/embedded-graphics/latest/embedded_graphics/\2#g
```

Comment:
These replacements shouldn't be necessary anymore, or are they still used somewhere?